src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/VacancySummariesProvider/VacancySummaryAggQueryBuilder.cs

```diff
@@ -1,176 +1,166 @@
-                    { "status", 1 },
-                    { "appStatus", $"${ApplicationReviewLookupAliasName}.status" },
-                    { "legalEntityId", 1 },
-                    { "legalEntityName", 1 },
-                    { "employerAccountId", 1 },
-                    { "employerName", 1 },
-                    { "ukprn", $"$trainingProvider.ukprn" },
-                    { "createdDate", 1 },
-                    { "closingDate", 1 },
-                    { "startDate", 1 },
-                    { "closedDate", 1 },
-                    { "closureReason", 1 },
-                    { "applicationMethod", 1 },
-                    { "programmeId", 1 },
-                    { "duration", "$wage.duration" },
-                    { "durationUnit", "$wage.durationUnit" },
-                    { "transferInfoUkprn", "$transferInfo.ukprn" },
-                    { "transferInfoProviderName", "$transferInfo.providerName" },
-                    { "transferInfoTransferredDate", "$transferInfo.transferredDate" },
-                    { "transferInfoReason", "$transferInfo.reason" },
-                    { "trainingProviderName", "$trainingProvider.name" }
+                '$unwind': {
+                    'path': '$candidateApplicationReview',
+                    'preserveNullAndEmptyArrays': true
                 }
-            }
-        };
-        private static readonly BsonDocument _projectionTwoClause = new BsonDocument
-        {
+            },
             {
-                "$project",
-                new BsonDocument
-                {
-                    { "vacancyGuid", 1 },
-                    { "vacancyReference", 1 },
-                    { "title", 1 },
-                    { "status", 1 },
-                    { "legalEntityId", 1 },
-                    { "legalEntityName", 1 },
-                    { "employerAccountId", 1 },
-                    { "employerName", 1 },
-                    { "ukprn", 1 },
-                    { "createdDate", 1 },
-                    { "closingDate", 1 },
-                    { "startDate", 1 },
-                    { "closedDate", 1 },
-                    { "closureReason", 1 },
-                    { "applicationMethod", 1 },
-                    { "programmeId", 1 },
-                    { "duration", 1 },
-                    { "durationUnit", 1 },
-                    { "transferInfoUkprn", 1 },
-                    { "transferInfoProviderName", 1 },
-                    { "transferInfoTransferredDate", 1 },
-                    { "transferInfoReason", 1 },
-                    { "trainingProviderName", 1 }
+                '$project': {
+                    'vacancyGuid': '$_id',
+                    'vacancyReference': 1,
+                    'title': 1,
+                    'status': 1,
+                    'appStatus': '$candidateApplicationReview.status',
+                    'appIsWithdrawn': '$candidateApplicationReview.isWithdrawn',
```

Comment:
Wouldn't it be easier if we have a match clause before this one to filter out withdrawn applications?

Reply:
> The bigger change is I've simplified `VacancySummaryAggQueryBuilder` so it just loads the entire aggregate pipeline from a string rather than composing it using a series of BsonDocuments and BsonArrays. 

Works for me. 👍

Reply:
@dev4valtech Yes it would and we could do this using a more recent version of mongo.  The Cosmos DB version doesn't allow you to add a match clause in the $lookup.

Reply:
Why can't you use another match stage in the pipeline after to filter it out